src/WordPress/JsonMapper/ArrayInformation.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace WordPress\JsonMapper;
+
+class ArrayInformation implements \JsonSerializable
```

Comment:
Does it need to be JsonSerializable?

Reply:
Actually, this class seems to be just a container for two values: `isArray` and `dimensions`. Could it be refactored into either a simple array with two keys, or a simple class with two public properties and no methods?

Reply:
I agree. I'll see if it is truly needed at all and how to implement it in the simplest way possible.

Reply:
Gone. Gone like the wind.